.env

```diff
@@ -1,8 +0,0 @@
-# This is an example where you can store your environment variables.
```

Comment:
We want to ignore this file so it can be modified locally and not tracked changes